### ISSUE TITLE
[1.19.2] Client crash upon login. (ConcurrentModificationException)

### ISSUE THREAD

**Issue by Sandriell** [refinedmods/refinedstorage#3574]
### Describe the bug

While logging into a dedicated server, the client crashed. A subsequent login worked fine however, so maybe a rare race condition?

### How can we reproduce this bug or crash?

Unsure, only occurred once out of many logins, and then not again the next time.

### What Minecraft version is this happening on?

Minecraft 1.19.2

### What Forge version is this happening on?

43.2.21

### What Refined Storage version is this happening on?

1.11.6

### Relevant log output

```shell
https://gist.github.com/Sandriell/348e75e171ed0be429b799f89f25f0bb
```

**Comment by raoulvdberge** [comment]
Sorry, 1.19.2 is no longer maintained.

**Comment by Sandriell** [comment]
> Sorry, 1.19.2 is no longer maintained.

You just released a 1.19.2 update 2 days ago.

**Comment by raoulvdberge** [comment]
Only stupid people never change their minds :-)
That 1.19.2 release was mainly created to fix an important (world crashing) issue (and to backport the other bugfixes at that point because why not)
